### ISSUE TITLE
Add a setting to disable "Social Title"

### ISSUE THREAD

**Issue by darkf3d3** [PrestaShop/PrestaShop#23914]
I think that Prestashop need to implement into back office a setting to disable completely social title.

**Specs**
A toggle is displayed on customer setting page.
The toggle label: 'Ask for customers\' title' in _Admin.Shopparameters.Feature_
 The toggle is enabled by default. 
When the option is disabled, the social title are not displayed on the checkout process in the front office.

**Comment by florine2623** [comment]
Ping @PrestaShop/product-team ,

Is a feature to disable Customer social titles possible ? 
![Capture d’écran 2021-04-07 à 11 09 06](https://user-images.githubusercontent.com/16019289/113841468-0764d480-9792-11eb-9a58-ccd83e901470.png)

Thanks!

**Comment by alfonsolr09** [comment]
yes, you can delete the treatments and they will not appear

**Comment by darkf3d3** [comment]
Yes, I saw that's possible to delete them, but I think that could be useful having the possibility to show/hide them from front office.
For example If you just want to stop asking for them but keep the datas of already registered customer, or simply bacause I case you decide to enable them again you you don't have to create new social titles.

**Comment by alfonsolr09** [comment]
Cierto

**Comment by MatShir** [comment]
I suggest adding the option right below the list of Titles.
I will be a toggle: "Enable social title" enabled by default.

**Comment by MatShir** [comment]
@Julievrz Could you check what would be the best wording for this feature ? :)

**Comment by Julievrz** [comment]
> @Julievrz Could you check what would be the best wording for this feature? :)

I suggest 'Ask for customers' title' in Admin.Shopparameters.Feature
